docs/kubernetes/user-guides/driver-install.md

```diff
@@ -1,22 +1,45 @@
 # Kubernetes Driver Installation Guide
 
+## GCP Permissions Required
+
+The service account performing driver installation must have at least the
+following GCP IAM permissions:
+
+```
+resourcemanager.projects.getIamPolicy
```

Comment:
This is to run the initial setup script that creates a new service account right?

If a user wants to import/use an existing service account, then the set of permissions they need would be different.